OCPBUGS-17994: Note about Kubernetes-NMState and br-ex is too specific (#68187)
openshift/openshift-docs: modules/virt-about-nmstate.adoc

[NOTE]
====

> If your {product-title} cluster uses OVN-Kubernetes as the network plugin, configuration changes to the interface is not allowed. As a workaround, use a secondary network interface connected to your host or switch to the OpenShift SDN network plugin.

Reviewer: is>are since it refers to the plural changes earlier in the sentence:
```suggestion
If your {product-title} cluster uses OVN-Kubernetes as the network plugin, configuration changes to the interface are not allowed. As a workaround, use a secondary network interface connected to your host or switch to the OpenShift SDN network plugin.
```

> If your {product-title} cluster uses OVN-Kubernetes as the network plugin, configuration changes to the interface is not allowed. As a workaround, use a secondary network interface connected to your host or switch to the OpenShift SDN network plugin.

Reviewer: Small suggestion to write this with more user focus and to use active voice:
```suggestion
If your {product-title} cluster uses OVN-Kubernetes as the network plugin, you cannot make configuration changes to the interface. As a workaround, use a secondary network interface connected to your host or switch to the OpenShift SDN network plugin.
```

Totally optional and up to you which wording you like better.
